Patch parameters and other changes. (#64)
campa-consortium/pals: README.md

# Particle Accelerator Lattice Standard (PALS)

An effort to create a standard to promote lattice information exchange for particle accelerators.

> PALS can also serve as the language for new simulation programs and 

Reviewer: Language seems a bit vague here, how about:
```suggestion
PALS can also serve as the beamline scheme for new simulation programs and 
```

Reviewer: I think language format is not a commonly used name, the computer science term of what we do is schema or markup.

Author: Common or not, I find `language format` to be clearer than `schema` or `markup`

Reviewer: Thanks, agreed.

Following these definitions, I think **schema** or **data model** is what we define here.

the associated code that comes

> with PALS will be usable to read/write lattice files thus saving considerable programming time.

Reviewer: Validation is also very useful:
```suggestion
with PALS will be usable to read/validate/write lattice files thus saving considerable programming time.
```

Author: Changed.

[Online documents.](https://pals-project.readthedocs.io)

## Scope

> PALS is a standard for describing  focus on the physical layout and properties of a machine.

Reviewer: This sentence seems incomplete.
```suggestion
PALS is a standard for describing particle accelerator lattices, with focus on the physical layout and properties of a machine.
```

Author: Changed.

> PALS is a standard for describing  focus on the physical layout and properties of a machine.

Author: Changed.

The tracking of particles is not part of PALS although some tracking related parameters that
can be useful to share will be described by the standard.

## Participation

Community involvement is invited! 
There are weekly (more or less) meetings to discuss development of the lattice standard.
To participate in the creation of the lattice standard, 
please contact [Jean-Luc Vay](https://github.com/jlvay) to be put on the mailing list. 

## Goals

1. **Community outreach:** Through papers, talks and other avenues, inform the accelerator community
about PALS. Additionally, seek involvement from the community.

> 2. **Develop the PALS standard:** 

Reviewer: Missing content after the `:`?

Author: Replaced with period.

> 3. **[Develop parsing code to read/write/manipulate PALS lattices](https://pals-project.readthedocs.io/en/latest/lattice.html)**

Reviewer: ```suggestion
3. **[Develop user-friendly libraries & tools to read/write/manipulate/visualize PALS lattices](https://pals-project.readthedocs.io/en/latest/lattice.html)**
```
